components/builder-worker/src/runner/mod.rs

```diff
@@ -34,7 +34,10 @@ use bldr_core::logger::Logger;
 use chrono::Utc;
 use hab_core::os::users;
 use hab_core::package::archive::PackageArchive;
-use hab_core::util::perm;
+#[cfg(not(windows))]
+use hab_core::util::posix_perm as perm;
+#[cfg(windows)]
```

Comment:
What happens if you leave the `windows` include out? Since this isn't currently built for windows, I think it should work.

Reply:
It builds and runs fine for me, but of course I'm running it on Linux so it should. I didn't know that this wasn't built for windows. I can totally remove it if it's not necessary.